test/e2e/local.sh

```diff
@@ -18,7 +18,7 @@ source "$(dirname "$0")/../common.sh"
 source "$(dirname "$0")/setup.sh"
 
 export KIND_CLUSTER="local-kubebuilder-e2e"
-create_cluster ${KIND_K8S_VERSION:-v1.18.15}
+create_cluster ${KIND_K8S_VERSION:-v1.22.1}
 if [ -z "${SKIP_KIND_CLEANUP:-}" ]; then
```

Comment:
That is very nice. Only the code changes do not close the issue. 
We need to make the tests get done by prow. @estroz do you remember the repo?